packages/dashboard/src/components/datagrid/DataGrid.tsx

```
@@ -105,16 +104,34 @@ export default function DataGrid<TRow extends DataGridRowType = DataGridRow>({
+    () => columns.map((c) => c.key),
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [columns.map((c) => c.key).join(',')]
+  );
+
+  const { columnKeys, reorderColumns } = useColumnOrder(
+    storageKey ?? '',
+    defaultColumnKeys
+  );
+
+  const orderedColumns = useMemo(() => {
+    if (!storageKey) return columns;
```

ESLint

[error] Expected { after 'if' condition.

(curly)